test.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Let's remove this file. Doesn't seem to be relevant for this PR.

Reply:
All of these were for automation. But, I will remove them